linux/deploy_vm/rebuild_unattend_install_iso.yml

```diff
@@ -62,11 +100,14 @@
             dest_file: "/boot/grub/grub.cfg"
           - src_file: "{{ new_unattend_install_conf }}"
             dest_file: "/preseed/ubuntu.seed"
+          - src_file: "{{ src_iso_file_dir }}/isolinux.cfg"
+            dest_file: "/isolinux/isolinux.cfg"
+      when: not result_extract_file.failed
```

Comment:
Line 94 ~ 104 can be inserted before line 81 since they have same when condition.

Reply:
Change it. Thanks